checkov/common/runners/runner_registry.py

```diff
@@ -281,6 +280,7 @@ def _handle_report(self, scan_report: Report, repo_root_for_plan_enrichment: lis
             scan_report = Report("terraform_plan").enrich_plan_report(scan_report, enriched_resources)
             scan_report = Report("terraform_plan").handle_skipped_checks(scan_report, enriched_resources)
         self.scan_reports.append(scan_report)
+        integration_feature_registry.run_post_runner(scan_report)
```

Comment:
@lirshindalman  shouldn't it happen before appending the report?

Reply:
you are right